modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFormProvider.java

```diff
@@ -72,10 +73,21 @@ public InfoForm getInfoForm() {
 	public InfoForm getInfoForm(JournalArticle article) {
 		DDMStructure ddmStructure = article.getDDMStructure();
 
+		AssetEntry assetEntry = _assetEntryLocalService.fetchEntry(
+			JournalArticle.class.getName(), article.getResourcePrimKey());
+
 		long ddmStructureId = ddmStructure.getStructureId();
 
 		try {
-			return getInfoForm(ddmStructureId);
+			InfoForm infoForm = getInfoForm(ddmStructureId);
+
+			if (assetEntry != null) {
```

Comment:
I think it's important to think through under what circumstances it would be null, and what we want to do in those cases. Otherwise we risk hiding a larger problem by just not showing a few fields.